docs/cse/integrations/configuring-threatq-source-in-cse.md

```diff
@@ -7,55 +7,11 @@ description: Learn how to set up a ThreatQ source.
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-This topic has information about configuring a ThreatQ source in Cloud SIEM.
+ThreatQ is a threat intelligence platform that centrally manages and correlates external sources of threat intel information. If you have a ThreatQ subscription, you can leverage ThreatQ threat intel feeds. 
 
-ThreatQ is a threat intelligence platform that centrally manages and correlates external sources of threat intel information. 
+To do so, [ingest threat intelligence indicators](/docs/platform-services/threat-intelligence/about-threat-intelligence/#ingest-threat-intelligence-indicators) from your ThreatQ source.
```

Comment:
Is this ingestion something that the user would need to set up?

Reply:
Yes. That link should tell them what they need to know.